sim/battle.js

```diff
@@ -2348,7 +2737,7 @@ class Battle extends Dex.ModdedDex {
 		case 'instaswitch':
 		case 'switch':
 			if (decision.choice === 'switch' && decision.pokemon.status && this.data.Abilities.naturalcure) {
-				this.singleEvent('CheckShow', this.data.Abilities.naturalcure, null, decision.pokemon);
+				this.singleEvent('CheckShow', this.getAbility('naturalcurse'), null, decision.pokemon);
```

Comment:
> `naturalcurse`

Reply:
(For the record, this is one reason why I obsessively double-click and copy/paste even when retyping would be faster.)